app/gen-server/lib/HomeDBManager.ts

```diff
@@ -2397,7 +2397,9 @@ export class HomeDBManager extends EventEmitter {
     .leftJoinAndSelect('workspace_group_users.logins', 'workspace_user_logins')
     // Join the org and groups/users.
     .leftJoinAndSelect('workspaces.org', 'org')
-    .leftJoinAndSelect('org.aclRules', 'org_acl_rules')
+    .leftJoinAndSelect('org.aclRules', 'org_acl_rules',
+      '("org_acl_rules"."permissions" <> "acl_rules"."permissions")'
```

Comment:
Can you remind me if a condition placed here is implemented using ON or WHERE?

Reply:
Is there an easy way to convince myself this will have no impact on the `parentAccess` or `orgAccess` fields in `UserAccessData`?

Reply:
After seing your comment, I tried to justify myself, but the more I did, the more I felt a cognitive dissonance.

I instead propose another approach (originally suggested by Vincent) which I hope is less risky and also improves the code of this function: #824

If you also feel more comfortable with this other PR, I would be happy to close this one!